backend/src/Examples/ExampleApp.Examples/Handlers/Booking/Reservations/MyReservationsBase.cs

```diff
@@ -0,0 +1,42 @@
+using ExampleApp.Examples.Contracts.Booking;
+using ExampleApp.Examples.Contracts.Booking.Reservations;
+using ExampleApp.Examples.DataAccess;
+using Microsoft.AspNetCore.Http;
+
+namespace ExampleApp.Examples.Handlers.Booking.Reservations;
+
+public class MyReservationsBase(ExamplesDbContext dbContext)
+{
+    protected IQueryable<MyReservationDTO> MyReservations(HttpContext context)
+    {
+        var customerId = context.GetCustomerId();
+        return dbContext
+            .Reservations.Where(r => r.CustomerId == customerId)
+            .Join(
+                dbContext.Timeslots,
+                reservation => reservation.TimeslotId,
+                timeslot => timeslot.Id,
+                (reservation, timeslot) => new { reservation, timeslot }
```

Comment:
```suggestion
                r => r.TimeslotId,
                t => t.Id,
                (r, t) => new { Reservation = r, Timeslot = t }
```

Do we not have some IDE rule that forbids declaring properties with non-PascalCasing?

Reply:
It does not apply to anonymous types (and that is a very sensible approach IMO, otherwise the anonymous types start to be awkward to use in some contexts).

Reply:
Renamed